### ISSUE TITLE
Outlook link not working

### ISSUE THREAD

**Issue by LucienPeng** [barinbritva/add-to-calendar#13]
Hello! 

I was finding the solution for adding events to calendars and I think this pacakage is pretty cool! 

However, I noticed that the outlook link seems to be not working, the link opened will be redirected to outlook homepage. 

It would be nice if this could be fixed!

Thank you

**Comment by barinbritva** [comment]
Hi LucienPeng,

Thanks for your feedback!

Here is a link generated via the lib and it works fine for me. Could you please check if it works for you?
`https://outlook.live.com/calendar/0/deeplink/compose?path=%2Fcalendar%2Faction%2Fcompose&rru=addevent&subject=Ali%20Ragimov%20and%20Ali%20Ragimov&startdt=2024-02-20T07%3A00%3A00Z&enddt=2024-02-20T07%3A15%3A00Z`

Please attach your result link to me be able to check it.
As you mentioned you get redirected to the home page, the reason is you probably are not signed in to outlook.com at the moment.

**Comment by LucienPeng** [comment]
Hello barinbritva, 

Thanks for your quick response! 
Here's the result link :
 [https://www.microsoft.com/zh-tw/microsoft-365/outlook/email-and-calendar-software-microsoft-outlook?deeplink=%2fowa%2f0%2f%3fpath%3d%252fcalendar%252faction%252fcompose%26rru%3daddevent%26state%3d1%26redirectTo%3daHR0cHM6Ly9vdXRsb29rLmxpdmUuY29tL2NhbGVuZGFyLzAvZGVlcGxpbmsvY29tcG9zZS8_cGF0aD0lMmZjYWxlbmRhciUyZmFjdGlvbiUyZmNvbXBvc2UmcnJ1PWFkZGV2ZW50JnN1YmplY3Q9QWxpK1JhZ2ltb3YrYW5kK0FsaStSYWdpbW92JnN0YXJ0ZHQ9MjAyNC0wMi0yMFQwNyUzYTAwJTNhMDBaJmVuZGR0PTIwMjQtMDItMjBUMDclM2ExNSUzYTAwWg&sdf=0](url)
 
 I was signned in by the time. But it was my work account which has some strict restrictions.
 If you're able to see it, then I believe it's my own problem. 😅 I'll see with my team if that's the case.
 
 Thanks again for your time and your reply!

**Comment by barinbritva** [comment]
I can see your link contains language code `www.microsoft.com/zh-tw/microsoft-365...` which looks like you copied it already from redirect. But I need a link you get from the lib. The correct link should start from `https://outlook.office.com/calendar/0/deeplink/compose` or `https://outlook.live.com/calendar/0/deeplink/compose`

And here is the catch, btw. Unfortunately, Outlook hosts work and personal accounts in different domains. So, to generate link for personal calendars you should use `new Outlook()`, but for work calendars `new Office365()`.

Probably it's your case


```typescript
const event = ....
const personalLink = new Outlook().createLink(event)
const workLink = new Outlook().createLink(event)

console.log(personalLink, personalLink);
```

Just generate both links and then put them to browser. One of them should works

Let me know

**Comment by LucienPeng** [comment]
Hello brainbritva, 

This is the llnk from lib :

`https://outlook.live.com/calendar/0/deeplink/compose?path=%2Fcalendar%2Faction%2Fcompose&rru=addevent&subject=DS%202024-02-07%2013%3A08&startdt=2024-03-31T11%3A08%3A49Z&enddt=2024-03-31T11%3A09%3A49Z&body=%3Cb%3ETest%3C%2Fb%3E%3C%2Fbr%3E%3C%2Fbr%3E%3Ca%20href%3Dundefined%3Eundefined%3C%2Fa%3E&location=&to=`

I tried with office365 but the page loads all the time, so I guess it's also not work. 😔
`https://outlook.office.com/calendar/0/deeplink/compose?path=%2Fcalendar%2Faction%2Fcompose&rru=addevent&subject=DS%202024-02-07%2013%3A08&startdt=2024-03-31T11%3A08%3A49Z&enddt=2024-03-31T11%3A09%3A49Z&body=%3Cb%3ETest%3C%2Fb%3E%3C%2Fbr%3E%3C%2Fbr%3E%3Ca%20href%3Dundefined%3Eundefined%3C%2Fa%3E&location=&to=`


in addition, is there anyway to add event to Apple calendar? 

Thank you for your reply!

**Comment by barinbritva** [comment]
Hi LucienPeng,

Thanks for your feedback. It seems that the redirect for Office 365 doesn't work because there `/0/` part. I removed it. Now it should work.

Please try:
`npm install --save @barinbritva/add-to-calendar@next`
If it works I will release the main branch

Speaking of Apple, there is no web interface to pass params to. For Apple mail app you can generate an `ics` file and attach it. The mail app recognze such kind of attachments and shows the UI to add an event to calendar. It work for all popular mail clients

Looking forward to hear from you

**Comment by rapkaan** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
clint419, @reeya2022, @pcaversaccio, @samburgers, @0xArsen, @yihau, @achiko, @robert197, @scottrepreneur, @Enadjaaou, @Kerandor, @gat786, @RuggeroCino, @benbaessler, @pistomat, @lanastasov, @robert-e-davidson3, @Binb1, @b17z, @andy-hook

**Comment by barinbritva** [comment]
@LucienPeng hi,

Is the issue still relevant for you? Let me know if you resolved your issue or found another library.

Just in case I released version 1.3.2 with the fix.

**Comment by barinbritva** [comment]
Fixed.
